sdk/core/azure-core/test/ut/operation_test.hpp

```diff
@@ -81,6 +86,15 @@ namespace Azure { namespace Core { namespace Test {
     // This is a helper method to allow testing of the underlying operation<T> behaviors
     //  ClientOperations would not expose a way to control status
     void SetOperationStatus(OperationStatus status) { m_status = status; }
+
+    static StringOperation CreateFromResumeToken(
+        std::string const& resumeToken,
+        StringClient const& client)
+    {
+      StringOperation operation(client, resumeToken);
```

Comment:
nit: to avoid coding pattern confusion when people look at code samples in tests, swap the parameters